shepherd/LCM.py

```diff
@@ -0,0 +1,49 @@
+import lcm
+import threading
+import json
+
+class LCMClass:
```

Comment:
Add a comment here describing how this class works very generally. Also describe how to make use of the LCM object.